plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/common/ColorField.tsx

```diff
@@ -0,0 +1,28 @@
+import { styled } from "@web/theme";
+
+import { TextInput } from "./styled";
+
+const ColorField: React.FC<{ color?: string; value?: string }> = ({ color, value }) => {
+  return (
+    <>
+      <ColorBlock color={color ?? ""} />
+      <TextInput value={value ?? ""} />
+    </>
```

Comment:
Since this is a color "field", is it okay (does it make sense) to include in this the Field component instead of wrapping in a fragment? Either is okay I suppose